src/platforms/native/guides/minidumps/index.mdx

```diff
@@ -116,7 +116,7 @@ documents.
 ## Event Attachments
 
 Besides the minidump file, Sentry can optionally store additional files uploaded
-in the same request, such as log files. Learn more about these in [Attachments](enriching-error-data/attachments/).
```

Comment:
cc @jan-auer, this page is disabled for minidumps, see https://github.com/getsentry/sentry-docs/pull/2343

Reply:
I tried to re-enable it on Friday, but Gatsby wouldn't let me. Maybe you can help me with that? I'd like to keep the link in.